pkgs/build-support/flatten-references-graph/default.nix

```diff
@@ -0,0 +1,29 @@
+{
+  bash,
+  nix-gitignore,
+  python39Packages,
```

Comment:
Do you have to pick Python 3.9 or would our currenty default Python3 also work here?

Reply:
python3 works, not sure why I went with 39 tbh
fixed in 9749846ede1c4eba2c404332a57837e8589bce2b